src/Jackett.Common/Definitions/animeworld.yml

```
@@ -1,47 +1,32 @@
 ---
 id: animeworld
-name: AnimeWorld
-description: "AnimeWorld (AST4u) is a GERMAN Private site for TV / MOVIES / ANIME / HENTAI"
+name: Animeworld
```

```suggestion
name: AnimeWorld
```
Was this changed by mistake?

---

Yes it was, sorry about that, didn't catch it